docs/devguide/changelog.rst

```diff
@@ -2,6 +2,137 @@ Changelog
 =========
 
 
+Parsl 1.0.0
+-----------
+
+Released on June <FILL>th, 2020
+
+Parsl v1.0.0 includes 57 closed issues and 232 pull requests with contributions (code, tests, reviews and reports) from:
+
+Akila Ravihansa Perera @ravihansa3000, Aymen Alsaadi @AymenFJA, Anna Woodard @annawoodard,
+Ben Clifford @benclifford, Ben Glick @benhg, Benjamin Tovar @btovar, Daniel S. Katz @danielskatz,
+Daniel Smith @dgasmith, Douglas Thain @dthain, Eric Jonas @ericmjonas, Geoffrey Lentner @glentner,
+Ian Foster @ianfoster, Kalpani Ranasinghe @kalpanibhagya, Lindsey Gray @lgray, Logan Ward @WardLT,
+Lyle Hayhurst @lhayhurst, Mihael Hategan @hategan, Rajini Wijayawardana @rajiniw95, @saktar-unr,
+Tim Shaffer @trshaffer, Tom Glanzman @TomGlanzman, Yadu Nand Babuji @yadudoc and,
+Zhuozhao Li @ZhuozhaoLi
+
+New Functionality
+^^^^^^^^^^^^^^^^^
+
+* **Deprecated** and removed features:
```

Comment:
The first thing under new functionality is the stuff we don't want you to use?  Maybe make deprecated a top level heading, then new functionality as another top level header after this section.

Reply:
I've split this out into a separate heading. I think this needs to be at the top as a warning.